src/java/org/apache/cassandra/io/util/FileUtils.java

```diff
@@ -66,7 +66,7 @@
 import static org.apache.cassandra.utils.Throwables.maybeFail;
 import static org.apache.cassandra.utils.Throwables.merge;
 
-public final class FileUtils
+public class FileUtils
```

Comment:
It does not matter to have it final as it already has private constructor so it can not be extended anyway as extender can not call super and it just blocks mocking as it can not deal with finals.

Reply:
not relevant anymore